### PR TITLE
chore: adding back DMSans-Bold.ttf used in SVGs

### PR DESCRIPTION
**Pull request by nicolasburtey**

### CODE REVIEW

File: app/rne-theme/theme.ts [hunk 1]
```diff
@@ -38,7 +38,9 @@ const theme = createTheme({
     Text: (props, theme) => {
       const universalStyle = {
         color: props.color || theme.colors.grey5,
-        fontFamily: "SourceSansPro",
```
Comment (UncleSamtoshi) [inline app/rne-theme/theme.ts:41]: Why was this removed?

**Review by UncleSamtoshi — CHANGES_REQUESTED** [review]

File: app/rne-theme/theme.ts (hunk 1 above)
Comment (nicolasburtey) [reply app/rne-theme/theme.ts:41]: red screen of death complaining the font doesn't exist. tried adding `-Regular` but got same message.

Comment (nicolasburtey) [reply app/rne-theme/theme.ts:41]: here is the issue, with the previous code: 

```
        fontWeight: props.bold ? "600" : "400",
        fontFamily: "SourceSansPro-Regular",
```

image

Comment (nicolasburtey) [reply app/rne-theme/theme.ts:41]: > here is the issue, with the previous code:
> 
> ```
>         fontWeight: props.bold ? "600" : "400",
>         fontFamily: "SourceSansPro-Regular",
> ```
> 
> image

@UncleSamtoshi if it's not been used in the app but only storybook I propose you to approve so we can unlock the release and can dive into this whenever we use the component

Comment (UncleSamtoshi) [reply app/rne-theme/theme.ts:41]: Works for me

**Review by UncleSamtoshi — APPROVED** [review]
